### DISCUSSION TITLE
Channels as a foundation for multi vendor marketplace

### DISCUSSION THREAD

**Discussion by nicolasleon** [saleor/saleor#8231] [Q&A]
With the new channels feature baked into v3 can Saleor be used to build a multi-vendor marketplace?

**Comment by karolkielecki** [comment]
@nicolasleon Hi, a multi-vendor and a multi-channel are two different features. Using the multi-channel feature, you can manage many sales channels from a single dashboard. 

Regarding a marketplace, there is an integration with Jetti: https://jetti.io/integrations/store/saleor

**Reply by basilisSam** [reply]
Hello, so with some modification of the code we can make it works like multi-vendor ?

**Reply by karolkielecki** [reply]
@basilisSam You will need to modify the entire Saleor. Just "some modification" won't be enough.
